tests/e2e/vmc_delete_tkg.go

```diff
@@ -0,0 +1,256 @@
+
+		scParameters[svStorageClassName] = storagePolicyName
+		storageclass, err := createStorageClass(clientNewGc, scParameters, nil, "", "", false, "")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		pvclaim, err := createPVC(clientNewGc, namespaceNewGC, nil, "", storageclass, "")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		// Waiting for PVC to be bound.
+		var pvclaims []*v1.PersistentVolumeClaim
+		pvclaims = append(pvclaims, pvclaim)
+		ginkgo.By("Waiting for all claims to be in bound state")
+		persistentvolumes, err := fpv.WaitForPVClaimBoundPhase(clientNewGc, pvclaims, framework.ClaimProvisionTimeout)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		pv := persistentvolumes[0]
+		volHandle := getVolumeIDFromSupervisorCluster(pv.Spec.CSI.VolumeHandle)
+		gomega.Expect(volHandle).NotTo(gomega.BeEmpty())
+		svcPVCName := pv.Spec.CSI.VolumeHandle
+
+		defer func() {
+			err = clientNewGc.CoreV1().PersistentVolumeClaims(namespaceNewGC).Delete(ctx, pvclaim.Name, *metav1.NewDeleteOptions(0))
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			err = clientNewGc.StorageV1().StorageClasses().Delete(ctx, storageclass.Name, *metav1.NewDeleteOptions(0))
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}()
+
+		ginkgo.By("Creating a pod in new GC")
+		_, err = createPod(clientNewGc, namespaceNewGC, nil, pvclaims, false, execCommand)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		//just for testing
+		// err = fpod.DeletePodWithWait(clientNewGc, pod1)
+		// gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Delete TKG")
+
+		ginkgo.By("scaling up TKG worker node")
+		if vmcUser == "testuser" {
+			ginkgo.By("Get WCP session id")
+			gomega.Expect((e2eVSphere.Config.Global.VmcDevopsUser)).NotTo(gomega.BeEmpty(), "Devops user is not set")
+			wcpToken = getWCPSessionId(vmcWcpHost, e2eVSphere.Config.Global.VmcDevopsUser,
+				e2eVSphere.Config.Global.VmcDevopsPassword)
+			framework.Logf("vmcWcpHost %s", vmcWcpHost)
+		} else {
+			ginkgo.By("Get WCP session id")
+			gomega.Expect((e2eVSphere.Config.Global.VmcCloudUser)).NotTo(gomega.BeEmpty(), "Cloud user is not set")
+			framework.Logf("vmcWcpHost %s", vmcWcpHost)
+			wcpToken = getWCPSessionId(vmcWcpHost, e2eVSphere.Config.Global.VmcCloudUser,
+				e2eVSphere.Config.Global.VmcCloudPassword)
+		}
+		deleteTKG(vmcWcpHost, wcpToken, tkg_cluster)
+
+		err = getGC(vmcWcpHost, wcpToken, tkg_cluster)
+		gomega.Expect(err).To(gomega.HaveOccurred())
+
+		//validating static volume provisioning after gc deletion
+		volHandle = getVolumeIDFromSupervisorCluster(pv.Spec.CSI.VolumeHandle)
+		gomega.Expect(volHandle).NotTo(gomega.BeEmpty())
+
+		ginkgo.By("Creating namespace on existing GC")
+		ns, err = framework.CreateTestingNS(f.BaseName, client, map[string]string{
+			"e2e-framework": f.BaseName,
+		})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), "Error creating namespace on second GC")
```

Comment:
First GC right ?

"Error creating namespace on second GC" 

Change this error message